bot/cogs/admin.py

```diff
@@ -257,6 +257,26 @@ async def pog(self, ctx, *args):
             return
         await disp.WRONG_USAGE.send(ctx, ctx.command.name)
 
+    @commands.command()
+    @commands.guild_only()
+    async def fstart(self, ctx, *args):
```

Comment:
Let's just go with `=start` no?

Reply:
i have a slight preference to imply that this is an admin forcing a start, but don't mind either way realy

Reply:
Ok that's fine